rackcity/api/serializers/asset_serializers.py

```diff
@@ -16,7 +16,7 @@
 from rackcity.api.serializers.fields import RCIntegerField
 import copy
 
-from rackcity.models.model_utils import ModelType
+from ...utils.asset_changes_utils import get_changes_on_asset
```

Comment:
WHAT IS THIS IMPORT?! ... never before seen

Reply:
pycharm does it automatically iDK